site/content/docs/5.2/components/carousel.md

```diff
@@ -103,16 +103,30 @@ Carousel progress indicator is paused under multiple conditions:
 {{< added-in "5.2.0" >}}
 
 Adding a pause and play button is recommended to setup an accessible carousel.
-This `button` must immediately follow your carousel and have the custom `data-bs-control="play-button"` attribute. In addition, it must also have a `data-bs-target` attribute that matches the `id` of the `.carousel` element.
+When using this button, do not forget to turn off the pause on hover by adding `data-bs-pause="false"` to your carousel.
```

Comment:
You can activate both at the same time but the result will be almost unusable or weird at best. Indeed you will need to hover the slide to reach the pause button but hovering the slide will pause the carousel. In other words, you will have to pause the carousel in order to be able to pause it... Seems pretty useless to me... 🤔 

It can't be just an informative display since it's a clickable element. We can built an informative display but for accessibility purposes it must be something else, probably a  simple `span`.